src/components/navigation.tsx

```diff
@@ -43,17 +46,25 @@ export default function Navigation({
 
   return (
     <>
-      <header className="relative bg-white">
-        <nav aria-label="Top">
+      <div className="flex text-sm flex-row justify-between items-center">
+        <div>
+          <a href="http://www.colonialcollections.nl" className="text-grey-500">
+            Colonialcollections.nl
```

Comment:
Is this a reference to the parent site, not to the Dataset Browser?

Reply:
@Doppen ?

Reply:
Yes, to the parent site.

Reply:
1. Indeed
2. We can remove the link for now and add it later when the page is there.